velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1063,6 +1120,69 @@ bool isTableScanSupported(const TypePtr& type) {
   return true;
 }
 
+void JoinFuzzer::testCrossProduct(
+    const std::string& tableDir,
+    core::JoinType joinType,
+    const std::vector<std::string>& probeKeys,
+    const std::vector<std::string>& buildKeys,
+    const std::vector<RowVectorPtr>& probeInput,
+    const std::vector<RowVectorPtr>& buildInput) {
+  VELOX_CHECK(probeInput.size() > 0 && buildInput.size() > 0);
+  const auto probeType = asRowType(probeInput[0]->type());
+  const auto buildType = asRowType(buildInput[0]->type());
+  auto outputColumns =
```

Comment:
Can we generate the output columns as the other join type does?

Reply:
This is actually the same as how outputColumns is generated at line 1244 that is used for other join types. The code here is simpler because we already know the join type is not LeftSemiProjectJoin, LeftSemiFilterJoin, or AntiJoin. We only call testCrossProduct() when the join type is LeftJoin, FullJoin, or InnerJoin.